_posts/2025-06-04-reporte-ejecutivo-gerencia.md

```
@@ -0,0 +1,40 @@
+tags: [Ejecutivo, Asistente, Jefe comercial, Potencial]
+comments: false
+author: Catalina Pinilla
+---
+
+## Reporte de gestión comercial
+
+En esta sección se han reunido los elementos claves para que puedas identificar oportunidades de mejora, analizar cartera y tomar decisiones estratégicas basadas en tu desempeño.
+
+Los datos seleccionados en gestión comercial, irán directamente a este apartado mostrando tus índices
+
+#### Aquí te mostramos cómo se suman cada una de las facturas que gestiones a tu reporte.
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)